Cargo.toml

```
@@ -130,4 +130,5 @@ serde_json = "1.0.120"
 
 # other crates
 thiserror = "2.0.12"
+uuid = { version = "1.17.0" }
```

```suggestion
uuid = { version = "1.17.0", default-features = false; }
```

and then activate the features in the leaf crates as needed;

---

I can see the value of this suggestion - fine-control features at crate level to minimize dependencies. This is a separate work to go through and need to evaluate one-by-one  for all of our crates defined at workspace-level.

For this particular case of `uuid`, the default feature is `std` and will be needed for all crates. So I think enabling default feature at worksapce-level is perfectly fine.